go.mod

```diff
@@ -8,12 +8,11 @@ require (
 	github.com/coreos/pkg v0.0.0-20150728231633-7373797ee9cd
 	github.com/gorilla/websocket v0.0.0-20180201015256-4ac909741dfa
 	github.com/pquerna/cachecontrol v0.0.0-20160421231612-c97913dcbd76 // indirect
-	github.com/stretchr/testify v1.3.0 // indirect
-	golang.org/x/net v0.0.0-20190311183353-d8887717615a // indirect
 	golang.org/x/oauth2 v0.0.0-20190226205417-e64efc72b421
-	golang.org/x/text v0.3.1-0.20180807135948-17ff2d5776d2 // indirect
 	google.golang.org/genproto v0.0.0-20190508193815-b515fa19cec8 // indirect
 	google.golang.org/grpc v1.19.0
 	gopkg.in/square/go-jose.v2 v2.0.1 // indirect
-	gopkg.in/yaml.v2 v2.2.1
+	gopkg.in/yaml.v2 v2.2.2
+	k8s.io/component-base v0.0.0-20190627205834-327675bd8ec3
```

Comment:
Is `k8s.io/component-base` needed?  https://github.com/kubernetes/klog/blob/master/go.mod

Reply:
Guessing this is why the blow up.